cmd/ddev/cmd/dotddev_assets/commands/host/launch

```diff
@@ -2,18 +2,28 @@
 
 ## #ddev-generated
 ## Description: Launch a browser with the current site
-## Usage: launch [path]
-## Example: "ddev launch" or "ddev launch /admin/reports/status/php" or "ddev launch phpinfo.php"
+## Usage: launch [path|pma|mailhog]
+## Example: "ddev launch" or "ddev launch /admin/reports/status/php" or "ddev launch phpinfo.php" or "ddev launch pma" or "ddev launch mailhog"
 
-FULLURL=${DDEV_PRIMARY_URL}
-
-if [ -n "${1:-}" ] ; then 
-  if [[ ${1::1} != "/" ]] ; then 
-    FULLURL="${FULLURL}/"; 
-  fi
-
-  FULLURL="${FULLURL}${1}"; 
-fi
+case "${1}" in
+  mailhog)
```

Comment:
Perhaps we should use some kind of escaping character here to avoid conflicts with normal URLs

Reply:
if your webserver has a mailhog domain you could call it like `ddev launch /mailhog` so i think escape char is not really nessasary. but im open to it (if you find a good escape char)

here are the options i have considred:

a) `--` like `ddev launch --mailhog` this feels CLI like, but `-` is still valid in urls
b) add it as a Second parameter `ddev launch - mailhog` this would cleanly remove the conflict 
c) open for other ideas?